src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -4,23 +4,32 @@
 using Bit.Core.Repositories;
 using Bit.Core.Settings;
 using Dapper;
+using Microsoft.AspNetCore.DataProtection;
 using Microsoft.Data.SqlClient;
 
 namespace Bit.Infrastructure.Dapper.Repositories;
 
 public class UserRepository : Repository<User, Guid>, IUserRepository
 {
-    public UserRepository(GlobalSettings globalSettings)
+    private readonly IDataProtector _dataProtector;
+
+    public UserRepository(
+        GlobalSettings globalSettings,
+        IDataProtectionProvider dataProtectionProvider)
         : this(globalSettings.SqlServer.ConnectionString, globalSettings.SqlServer.ReadOnlyConnectionString)
-    { }
+    {
+        _dataProtector = dataProtectionProvider.CreateProtector("UserRepositoryProtection");
```

Comment:
❓ Do these go as constants anywhere?

Reply:
I am not aware of a central location that we would put something like this today. I think we could extract it to a `const` at the top of the class, as is customary, but will only have one reference in this class.

Reply:
I moved these to const fields within the class as a best practice.